Glint ember-animated/template-registry not found

**Describe the bug**
After upgrade ember-animated from `1.0.4` to `1.1.1`, and also follow up with apply glint usage changes as of [docs](https://ember-animation.github.io/ember-animated/docs#usage-with-glint), vscode typescript, `tsc --noEmit` or `glint` is reporting error for ember-animated's template registry as below.
```
error TS7016: Could not find a declaration file for module 'ember-animated/template-registry'. '<app name>/node_modules/.pnpm/.../node_modules/ember-animated/dist/template-registry.js' implicitly has an 'any' type.
  There are types at '<app name>/node_modules/ember-animated/declarations/template-registry.d.ts', but this result could not be resolved when respecting package.json "exports". The 'ember-animated' library may need to update its package.json or typings.

7 import type EmberAnimatedRegistry from 'ember-animated/template-registry'
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/addon/package.json
+++ b/addon/package.json
@@ -18,15 +18,15 @@
   "author": "Edward Faulkner <edward@eaf4.com>",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
+      "types": "./declarations/index.d.ts",
       "default": "./dist/index.js"
     },
     "./*": {
-      "types": "./dist/*.d.ts",
+      "types": "./declarations/*.d.ts",
       "default": "./dist/*"
     },
     "./test-support": {
-      "types": "./dist/test-support/index.d.ts",
+      "types": "./declarations/test-support/index.d.ts",
       "default": "./dist/test-support/index.js"
     },
     "./addon-main.js": "./addon-main.cjs"

